sql/localsql/nipost/poet_registration.go

```diff
@@ -56,16 +77,22 @@ func PoetRegistrations(db sql.Executor, nodeID types.NodeID) ([]PoETRegistration
 
 	dec := func(stmt *sql.Statement) bool {
 		registration := PoETRegistration{
-			Address:  stmt.ColumnText(1),
-			RoundID:  stmt.ColumnText(2),
-			RoundEnd: time.Unix(stmt.ColumnInt64(3), 0),
+			Address:  stmt.ColumnText(2),
+			RoundID:  stmt.ColumnText(3),
+			RoundEnd: time.Unix(stmt.ColumnInt64(4), 0),
 		}
-		stmt.ColumnBytes(0, registration.ChallengeHash[:])
+
+		nodeId := make([]byte, types.NodeIDSize)
+		stmt.ColumnBytes(0, nodeId)
+
+		registration.NodeId = types.BytesToNodeID(nodeId)
```

Comment:
NIT: `nodeID` is passed to the function and doesn't need to be read again from the DB:

```suggestion
		registration.NodeId = nodeID
```

`id` also doesn't need to be queried from the DB.